packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -420,7 +425,27 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
   }
 
   @autobind
-  private _onDropdownKeyDown(ev: React.KeyboardEvent<HTMLElement>) {
+  private _onDropdownBlur(ev: React.FocusEvent<HTMLDivElement>) {
+    if (this.state.isOpen) {
+      // Do not onBlur when the callout is opened
+      return;
+    }
+    if (this.props.onBlur) {
+      this.props.onBlur(ev);
+      if (ev.preventDefault) {
+        return;
+      }
```

Comment:
No, I mean the `return` if `preventDefault` is true. It will return in any case?